iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -634,68 +673,135 @@ static LogicalResult setRootConfigImpl(
   // Redirect to individual operations.
   auto setRootConfigFn = [&](Operation *op) -> LogicalResult {
     return TypeSwitch<Operation *, LogicalResult>(op)
-        .Case<linalg::Mmt4DOp, linalg::ContractionOpInterface,
-              IREE::LinalgExt::FftOp>([&](auto op) {
+        .Case<IREE::LinalgExt::FftOp, linalg::GenericOp, linalg::Mmt4DOp>(
+            [&](auto op) {
+              return setRootConfig(entryPointFn, op, tiledLoops);
+            })
+        .Case<linalg::ContractionOpInterface>([&](auto op) {
```

Comment:
Why split this into two cases?

Reply:
There is an ordering in the cases, the first match is tried first, then the second. We want the `mmt4d` to be picked up before `ContractionOp`, which should be picked up before `LinalgOp` defaults.

Reply:
Got you, thanks!